docs/getting-started.md

```diff
@@ -74,7 +74,83 @@ To create a new headless WordPress app using SnapWP, follow these steps:
 
 ### Manual Installation
 
-@todo
+1. Install the required packages in your Next.js project:
+
+    ```bash
+    npm install --save @snapwp/blocks @snapwp/core @snapwp/next @snapwp/query
+    ```
+
+2. Create an `.env` file in the project root. The contents for the .env file can be copied from `Dashboard > WPGraphQL > Settings > SnapWP Helper` from your [WordPress backend](#backend-setup).
```

Comment:
I would recommend 
```md
Dashboard > GraphQL > SnapWP
```
<img width="536" alt="image" src="https://github.com/user-attachments/assets/69b07d1e-aeec-4f79-b868-293c8534f9ca" />

I have attached a screenshot for flow reference. What do you think ?
It is a minor thing, but `Dashboard > WPGraphQL > Settings > SnapWP Helper` this makes me look twice even though I am sure I am doing the right thing.

Reply:
This feels like overkill, it isn't the focus of the section, and the previous screenshot already shows the menu options in context.